packages/io/3mf-serializer/package.json

```diff
@@ -34,8 +37,12 @@
     "fflate": "0.7.3"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^23.0.4",
+    "@rollup/plugin-node-resolve": "^15.0.1",
```

Comment:
Why does the 3mf-serializer depend on these two plugins, but most other packages don't?

Reply:
So, the 3mf serializer depends on a library called "", which is CommonJS (require based imports). Node of course understands both CommonJS and ES6 imports. But Rollup needs to find modules using plugin-node-resolve, and translate the CommonJS code to ES6 using plugin-commonjs.

Most of the packages depend on modeling and/or array-utils. These are both ES6 modules now. :)